### ISSUE TITLE
Websockets not working

### ISSUE THREAD

**Issue by dbackeus** [arunoda/meteor-ddp-analyzer#7]
I get this error in my JS console when opening my Meteor app using the proxy:

```
WebSocket connection to 'ws://localhost:3030/sockjs/788/lnfxrvrn/websocket' failed: Connection closed before receiving a handshake response
```

It then falls back to regular http calls.

If this is intentional it should be added to the readme.

**Comment by arunoda** [comment]
Yes. This is intentional and due to a limitation. But it won't affect your
in generally in development.
And never affect to the production deployment.

Send me a PR.

---

Arunoda Susiripala
I curate Meteor Weekly - Check it out!
http://meteorhacks.com/meteor-weekly/?utm_source=email-footer&utm_medium=email&utm_campaign=meteorweekly

**Comment by dbackeus** [comment]
What is the limitation you mentioned? Using localhost and not a proper domain?

**Comment by arunoda** [comment]
Actually DDP analyzer is a customized SockJS proxy. I have done some hacks
in order to get it working. But it prevents WebSockets. It can be fixed.
But that's not a high priority at the moment.

On Tuesday, August 12, 2014, David Backeus notifications@github.com wrote:

> What is the limitation you mentioned? Using localhost and not a proper
> domain?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/arunoda/meteor-ddp-analyzer/issues/7#issuecomment-51901475
> .

## 

---

Arunoda Susiripala

@arunoda http://twitter.com/arunoda
http://gplus.to/arunodahttps://github.com/arunoda
http://www.linkedin.com/in/arunoda

**Comment by dbackeus** [comment]
PR updating the README as a first step: https://github.com/arunoda/meteor-ddp-analyzer/pull/8
